README.md

```diff
@@ -1,2 +1,23 @@
 # SLURM-Singularity-bindings
```

Comment:
As this is pretty general now, let's rename? (Maybe in a different PR?)

```suggestion
# Batch-Singularity-bindings
```

Reply:
I'd say we better choose: "Batch scheduler Singularity bindings"